openmetadata-ui/src/main/resources/ui/src/components/AppContainer/AppContainer.tsx

```diff
@@ -55,6 +67,7 @@ const AppContainer = () => {
     if (currentUser?.id) {
       fetchLimitConfig();
     }
+    fetchApplicationList();
```

Comment:
@Kenil27 , I believe we should only fetch the applications if currentUser is available.